src/applications/vaos/new-appointment/components/TypeOfVisitPage.jsx

```diff
@@ -56,33 +58,23 @@ function TypeOfVisitPage({
           title="Type of visit"
           schema={schema}
           uiSchema={uiSchema}
-          onSubmit={() => routeToNextAppointmentPage(history, pageKey)}
-          onChange={newData => updateFormData(pageKey, uiSchema, newData)}
+          onSubmit={() =>
+            dispatch(routeToNextAppointmentPage(history, pageKey))
+          }
+          onChange={newData =>
+            dispatch(updateFormData(pageKey, uiSchema, newData))
+          }
           data={data}
         >
           <FormButtons
-            onBack={() => routeToPreviousAppointmentPage(history, pageKey)}
+            onBack={() =>
+              dispatch(routeToPreviousAppointmentPage(history, pageKey))
+            }
             pageChangeInProgress={pageChangeInProgress}
-            loadingText="Page change in progress"
```

Comment:
What was the reason for removing "in"?

Reply:
good catch, I must have removed it by accident. Putting it back in.